oximeter/db/src/lib.rs

```diff
@@ -32,13 +32,14 @@ use thiserror::Error;
 
 mod client;
 pub mod model;
-#[cfg(feature = "oxql")]
+#[cfg(any(feature = "oxql", test))]
 pub mod oxql;
 pub mod query;
+#[cfg(any(feature = "oxql", feature = "sql", test))]
+pub mod shells;
 #[cfg(any(feature = "sql", test))]
 pub mod sql;
 
-#[cfg(feature = "oxql")]
```

Comment:
It looks like we need to put this `#[cfg(feature = "oxql")]` back. If we build this create with `--no-default-features`, I expect this will fail as `OxqlResult` isn't defined in that case. Ditto for lines 144 and 149 below.

Reply:
hmm, this may be an outdated comment, it should be there. 

```rust
mod client;
pub mod model;
#[cfg(any(feature = "oxql", test))]
pub mod oxql;
pub mod query;
#[cfg(any(feature = "oxql", feature = "sql", test))]
pub mod shells;
#[cfg(any(feature = "sql", test))]
pub mod sql;

```

Reply:
Ok, sorry about that. I had some stashed comments from an earlier, unsubmitted review. My bad! As long as it works when building with `--no-default-features`, we're good :)

Reply:
sure does. I may add a build/cargo check step to test this in ci